cmd/zetaclientd/p2p_diagnostics.go

```diff
@@ -30,7 +30,7 @@ func RunDiagnostics(
 	startLogger zerolog.Logger,
 	peers p2p.AddrList,
 	hotkeyPk cryptotypes.PrivKey,
-	cfg config.Config,
+	cfg *config.Config,
```

Comment:
Wondering, are these changes from a merge? Using no pointer looks the correct approach and I think it's related to some recent codebase improvement

Reply:
https://github.com/zeta-chain/node/pull/2411/commits/4f61c380bd8629ea044d7284eabf99c09b447131